src/coreclr/vm/jitinterface.cpp

```diff
@@ -9323,6 +9323,475 @@ CorInfoTypeWithMod CEEInfo::getArgType (
+                            }
+                            else if (pMethodTable->GetNumInstanceFieldBytes() == 8)
+                                size |= STRUCT_SECOND_FIELD_SIZE_IS8;
+                            else
+                            {
+                                size = STRUCT_NO_FLOAT_FIELD;
+                            }
+                        }
+                        else if (pNativeFieldDescs->GetCategory() == NativeFieldCategory::FLOAT)
+                        {
+                            if (pNativeFieldDescs->NativeSize() == 4)
+                            {
+                                size = size & STRUCT_FLOAT_FIELD_FIRST ? (size ^ STRUCT_MERGE_FIRST_SECOND) : (size | STRUCT_FLOAT_FIELD_SECOND);
+                            }
+                            else if (pNativeFieldDescs->NativeSize() == 8)
+                            {
+                                size = size & STRUCT_FLOAT_FIELD_FIRST ? (size ^ STRUCT_MERGE_FIRST_SECOND_8) : (size | STRUCT_SECOND_FIELD_DOUBLE);
+                            }
+                            else
+                            {
+                                UNREACHABLE_MSG("Invalid NativeFieldCategory.----LoongArch64----3");
+                                //size = STRUCT_NO_FLOAT_FIELD;
+                            }
+                        }
+                        else
+                        {
+                            if (pNativeFieldDescs->GetNumElements() == 1)
+                            {
+                                fieldType = pNativeFieldDescs->GetFieldDesc()[0].GetFieldType();
+                                if (fieldType == ELEMENT_TYPE_R4)
+                                    size = size & STRUCT_FLOAT_FIELD_FIRST ? (size ^ STRUCT_MERGE_FIRST_SECOND) : (size | STRUCT_FLOAT_FIELD_SECOND);
+                                else if (fieldType == ELEMENT_TYPE_R8)
+                                    size = size & STRUCT_FLOAT_FIELD_FIRST ? (size ^ STRUCT_MERGE_FIRST_SECOND_8) : (size | STRUCT_SECOND_FIELD_DOUBLE);
+                                else if ((size & STRUCT_FLOAT_FIELD_FIRST) == 0)
+                                {
+                                    size = STRUCT_NO_FLOAT_FIELD;
+                                    goto _End_arg;
+                                }
+                                else if (pNativeFieldDescs->NativeSize() == 8)
+                                    size |= STRUCT_SECOND_FIELD_SIZE_IS8;
+                            }
+                            else
+                            {
+                                size = STRUCT_NO_FLOAT_FIELD;
+                            }
+                        }
+                        //else
+                        //{
+                        //    assert(!"------------should amend for LOONGARCH64!!!");
```

Comment:
Either delete or re-enable this commented out code.

Reply:
OK, Thanks